target/linux/lantiq/patches-5.4/0702-net-lantiq-enable-baby-jumbo-frames-on-xrx200.patch

```diff
@@ -0,0 +1,61 @@
+From 9fd8a972e4b9691797e62ce27687ab9f94881b2f Mon Sep 17 00:00:00 2001
```

Comment:
Patch for 5.4 differs from that for kernel 5.10. It's a mistake?

Reply:
Yes, the way MTUs are handled for DSA slave interfaces is different in these two versions.

In 5.4 max_mtu is hard-coded to ETH_MAX_MTU, and it's up to the user to set the mtu correctly.

5.10 has port_change_mtu and port_max_mtu callbacks and makes sure that the master interface has a big enough mtu for all the slave interfaces when the slave interface MTU is changed.